src/libawkward/array/RegularArray.cpp

```diff
@@ -1120,7 +1120,7 @@ namespace awkward {
     Index64 flathead = array.ravel();
     Index64 regular_flathead(flathead.length());
 
-    struct Error err = awkward_regulararray_getitem_next_array_regularize_64(
+    struct Error err = kernel::regulararray_getitem_next_array_regularize<int64_t>(
```

Comment:
Don't: the kernel name is `awkward_regulararray_getitem_next_array_regularize_64`.